app/workers/notifications/new_reaction_worker.rb

```diff
@@ -13,7 +13,14 @@ def perform(reaction_data, receiver_data)
       return unless %w[User Organization].include?(receiver_klass)
 
       receiver = receiver_klass.constantize.find_by(id: receiver_data.fetch(:id))
-      Notifications::Reactions::Send.call(reaction_data, receiver) if receiver
+      Notifications::Reactions::Send.call(reaction_data, receiver) if should_send?(receiver, receiver_klass)
+    end
+
+    def should_send?(receiver, receiver_klass)
+      return false unless receiver
+      return true if receiver_klass == "Organization"
+
+      !receiver.article_reaction_notifications
```

Comment:
I don't think you want the bang (!) here. By default this is true so if `receiver.article_reaction_notifications` is true then I would think `should_send?` would be true as well.

Reply:
Ah yes, you're right! Thanks.